install_files/ansible-base/roles/build-ossec-deb-pkg/tasks/main.yml

```diff
@@ -0,0 +1,138 @@
+---
+# Really this is a check to see if we have to create the repo folder,
+# if thats the case we are not under a local vagrant box with a synced
+# root folder.
+- name: Create src path folder
+  file:
+    state: directory
+    path: "{{ item }}"
+  with_items:
+    - "{{ repo_src_path }}"
+    - "{{ build_path }}"
+
+- name: Conditional copy over current directory for repo build
+  synchronize:
+    # All Debian package source trees are located in `install_files/<package>`
+    # in the root of the SecureDrop git repository.
+    src: ../../../../ossec-{{ purpose }}/
+    dest: "{{ repo_src_path }}"
+    # Ansible v1 does not support the `use_ssh_args`, which was added in 2.0.
+    # When resolving #1146, to upgrade Ansible, this option should be
+    # activated to permit builds in CI.
+    #   use_ssh_args: yes
```

Comment:
@msheiny Note this change. The role is now compatible with Ansible v1.8.4, but for #1666 you may need to edit this option. Made sure to preserve Ansible v1 support since #1146 is still open.